manifests/gcp_marketplace/chart/kubeflow-pipelines/values.yaml

```diff
@@ -0,0 +1,11 @@
+images:
+  mysql: mysql:5.6
+  minio: minio/minio:RELEASE.2018-02-09T22-40-05Z
+  apiServer: gcr.io/ml-pipeline/api-server:0.1.23
```

Comment:
Should we templatize the version here and make it the same as the version in `application.yaml`?